datafusion/core/src/scheduler/task.rs

```diff
@@ -128,7 +128,7 @@ impl Task {
         }
     }
 
-    /// Call [`Pipeline::poll_partition`], attempting to make progress on query execution
+    /// Call `Pipeline::poll_partition`, attempting to make progress on query execution
```

Comment:
It can be manually referenced. But I'm not sure whether `super` is suitable in this case
```suggestion
    /// Call [`Pipeline::poll_partition`][super::pipeline::Pipeline::poll_partition],
    /// attempting to make progress on query execution
```